quickjs/src/jniMain/kotlin/app/cash/quickjs/Zipline.kt

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2021 Square, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package app.cash.quickjs
+
+import app.cash.quickjs.internal.bridge.InboundService
+import app.cash.quickjs.internal.bridge.OutboundClientFactory
+import java.util.concurrent.Executors
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.asCoroutineDispatcher
+import kotlinx.coroutines.launch
+
+actual abstract class Zipline {
```

Comment:
Wondering if we should use this name in the API or just have it be the library and package name and use "InterfaceBridge" or something here instead.

Reply:
I really like brand names for entry points. Retrofit, Moshi, Timber, Guice.